NuGet/ChakraCore.Debugger.nuspec

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>ChakraCore.Debugger</id>
+    <version>$version$</version>
+    <description>Debugging companion library for the ChakraCore JavaScript engine.</description>
+    <authors>Microsoft</authors>
+    <projectUrl>https://github.com/Microsoft/ChakraCore-Debugger</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+  </metadata>
+
+  <files>
+    <file src="..\lib\Debugger.Service\ChakraDebugService.h" target="include" />
+    <file src="..\lib\Debugger.ProtocolHandler\ChakraDebugProtocolHandler.h" target="include" />
+
+    <file src="..\build\bin\x64\Debug\*" target="lib\x64\Debug" exclude="*.ilk" />
+    <file src="..\build\bin\x64\Release\*" target="lib\x64\Release" exclude="*.ilk" />
+    <file src="..\build\bin\Win32\Debug\*" target="lib\x86\Debug" exclude="*.ilk" />
+    <file src="..\build\bin\Win32\Release\*" target="lib\x86\Release" exclude="*.ilk" />
```

Comment:
Should it be "Win32" or "x86"?  Should "ARM" be included?

Reply:
Visual C++ has a legacy identifier for Windows/Desktop/x86: `Win32`, even when the platform name is `x86`. This is the expected output folder.

I'll add ARM.

Reply:
Done.